test/makeFlatWorld.ts

```diff
@@ -15,7 +15,7 @@ describe('make flat world', function () {
   for (const version of versions) {
     const registry = Registry('bedrock_' + version)
     const worldPath = join(__dirname, '/flat-world-' + version)
-    const World = PrismarineWorld(registry)
+    const World = PrismarineWorld('bedrock_' + version)
```

Comment:
This is not a correct change

Reply:
It requires string, not registry, ts compile was failing

Reply:
Ah, then this needs to be fixed in primarine-world